release-notes/2023-10/README.md

```diff
@@ -0,0 +1,252 @@
+
+<!-- Optional section to include comments and instructions needed to successfully upgrade from the previous release to the current release. If the only changes needed are already captured in Helm Chart Breaking Changes, this section is not needed. -->
+
+## New Features and Behavior changes
+
+- webserver 0.13.4
+    - Upgraded Redis dependency from 7.0 to 7.2. This is a breaking change. It is necessary to upgrade the entire cluster in parallel, which is not something Kubernetes will do automatically.
+    - Option #1: Set webserver.redis-cluster.redis.update-strategy.type = OnDelete
+        - Run the upgrade deployment
+        - Run kubectl -n <`namespace`> delete pods <`release`>-webserver-redis-0 <`release`>-webserver-redis-1 <`release`>-webserver-redis-2 <`release`>-webserver-redis-3 <`release`>-webserver-redis-4 <`release`>-webserver-redis-5
+        - The pods of the stateful set will be deleted and should be automatically recreated in parallel.
+        - Remove the overide of the redis update-strategy from the configuration. You can re-deploy to apply this change but it is not required.
+    - Option #2: Prior to upgrade, run: kubectl -n <`namespace`> delete statefulset <`release`>-webserver-redis
+        - This will delete the redis cluster, preventing UI access to the application.
+        - Now run the upgrade deployment. The redis cluster will be recreated and deployed in parallel.
+    - Once upgraded, Redis storage will be incompatibile with older versions of the software. If it is necessary to downgrade to an older version, you must perform a hard reset on the redis cluster:
+        - [Perform-a-hard-reset-on-the-redis-cluster.md](https://github.com/ni/install-systemlink-enterprise/tree/2023-10/release-notes/2023-10/Perform-a-hard-reset-on-the-redis-cluster.md)
+- dataframeservice 0.14.14
+    - The Dremio data set refresh job interval was increased from 2 minutes to 1 hour. This reduces overall load on Dremio.
+    - Customers are not required to uptake this change, but doing so will be beneficial as the net result in load reduction.
+    - The following steps cover what the customer needs to do:
+        - [How-to-reset-Dremio.md](https://github.com/ni/install-systemlink-enterprise/tree/2023-10/release-notes/2023-10/How-to-reset-Dremio.md)
+- dataframeservice 0.14.39
+    - The DataFrame Service now uses a more efficient method for writing data to new tables, replacing Kafka. The DataFrame Service will still use Kafka for data ingestion for tables created before the 2023-10 release, while tables created after upgrading to the 2023-10 release will have data written directly to S3 storage.
+    - After upgrading to the 2023-10 release, you can safely remove Kafka from your cluster once all pre-upgrade tables are set to readonly. Please note that disabling Kafka may lead to data loss if pre-upgrade tables are not readonly, because any buffered data may not get written to storage.
+    - To remove Kafka from the cluster:
```

Comment:
These  steps are long enough that this should be pulled into its own document and referenced here.

Reply:
Moved the steps to this file and linked to it from the release notes: 
- https://github.com/ni/install-systemlink-enterprise/blob/sle-release-notes-2023-10/release-notes/2023-10/Remove-Kafka-from-the-cluster.md